### PR TITLE
tileserver-gl integration

### PR DESCRIPTION
**Pull request by ImreSamu**
work in progress ( https://github.com/openmaptiles/openmaptiles/issues/122 )

- [x] `make start-tileserver`
- [x]  quickstart ` generate-metadata ./data/tiles.mbtiles`
- [x]  testing ..

now the  `"openmaptiles/openmaptiles-tools:latest"` ( https://github.com/openmaptiles/openmaptiles-tools/pull/10 )  contains the `generate-metadata` tool  , 
it is on the docker hub : https://hub.docker.com/r/openmaptiles/openmaptiles-tools/builds/

### CODE REVIEW

**Comment by stirringhalo** [comment]
Related issue: https://github.com/openmaptiles/openmaptiles/issues/36

**Comment by ImreSamu** [comment]
the current log  -after `make start-tileserver`

```log
dev:~/openmaptiles/openmaptiles$ make start-tileserver
 
***********************************************************
* 
* Download latest klokantech/tileserver-gl docker image    
* see documentation: https://github.com/klokantech/tileserver-gl
* 
***********************************************************
 
docker pull klokantech/tileserver-gl
Using default tag: latest
latest: Pulling from klokantech/tileserver-gl
ca67a6a92c6d: Pull complete 
b9763aaa8128: Pull complete 
ca9dcd41dfd3: Pull complete 
ddb4ed6275d2: Pull complete 
8b1865c81bfb: Pull complete 
7e2a404a32d9: Pull complete 
Digest: sha256:8c7701922cbc8374d7b8ea75a5edae6629236b009a322c207dd1849b169c08d1
Status: Downloaded newer image for klokantech/tileserver-gl:latest
 
***********************************************************
* 
* Start klokantech/tileserver-gl 
*       ----------------------------> check localhost:8080 
* 
***********************************************************
 
docker run -it -v $(pwd)/data:/data -p 8080:80 klokantech/tileserver-gl
Starting tileserver-gl v1.3.0-beta.1
No MBTiles specified, using tiles.mbtiles
Automatically creating config file for tiles.mbtiles
Run with --verbose to see the config file here.
Starting server
Listening at http://:::80/
```


I have added a big warning,but probably this is not enough  
```
* Start klokantech/tileserver-gl 
*       ----------------------------> check localhost:8080 
```

Because the last message from the tileserver **pointing to port ::80**
```
Starting server
Listening at http://:::80/
```

@lukasmartinelli : 
I am finished .. 
*  travis test OK! 
*  my local test OK! 

please re-test / review / modify / improve as you like ! 
( cc @klokan )

**Comment by ImreSamu** [comment]
the proposed new fast-start - for example for **switzerland**  -  only 4 lines  
```bash
git clone https://github.com/openmaptiles/openmaptiles.git
cd openmaptiles
./quickstart.sh switzerland
make start-tileserver
```

File: Makefile [hunk 1]
```diff
@@ -135,7 +135,7 @@ start-tileserver:
 	@echo "* "		
 	@echo "***********************************************************"
 	@echo " "
-	docker run -it -v $$(pwd)/data:/data -p 8080:80 klokantech/tileserver-gl
+	docker run -it --rm -v $$(pwd)/data:/data -p 8080:80 klokantech/tileserver-gl
```
Comment (stirringhalo) [inline Makefile]: Why the '$$'? I need to drop to a single '$' to run this manually in bash.

Comment (ImreSamu) [inline Makefile]: $$ ->  https://ubuntuforums.org/showthread.php?t=2146879

Comment (stirringhalo) [inline Makefile]: Thanks, makes sense as I was using it to test another mbtiles without a makefile!

**Comment by stirringhalo** [comment]
@lukasmartinelli Good to go?

**Comment by lukasmartinelli** [comment]
Nothing can break  :) You can merge these kinds of PRs.

**Pull request merged**